azurelinuxagent/agent.py

```diff
@@ -258,11 +292,12 @@ def usage():
     s += ("usage: {0} [-verbose] [-force] [-help] " # pylint: disable=C0103
            "-configuration-path:<path to configuration file>" # pylint: disable=C0330
            "-deprovision[+user]|-register-service|-version|-daemon|-start|"
-           "-run-exthandlers|-show-configuration]"
+           "-run-exthandlers|-show-configuration|-collect-logs [-mode:full]"
```

Comment:
I can't quite tell what's happening here; it seems you've added _just_ an open straight brace, and the corresponding close brace was already there. 

Do we now have mismatched braces? Or did we just have them before? I can't find the original open brace, so I'm leaning towards the second, but I just want to be sure!

Reply:
Looking at the current version of this code in develop, it seems like we did have mismatched braces (the closing one `]`). Not that big of a deal luckily because it was just an example string of various command-line options, but now it should be fixed.